shell/common/logging.cc

```diff
@@ -124,7 +172,19 @@ void InitElectronLogging(const base::CommandLine& command_line,
   // Don't resolve the log path unless we need to. Otherwise we leave an open
   // ALPC handle after sandbox lockdown on Windows.
   if ((logging_dest & LOG_TO_FILE) != 0) {
-    log_path = GetLogFileName(command_line);
+    if (filename_is_handle) {
+#if BUILDFLAG(IS_WIN)
+      // Child processes on Windows are provided a file handle if logging is
+      // enabled as sandboxed processes cannot open files.
+      log_handle = GetLogInheritedHandle(command_line);
+      if (!log_handle.is_valid()) {
+        DLOG(ERROR) << "Unable to initialize logging from handle.";
```

Comment:
Not having a handle in this case means the child process could not log which seems like a useful error to surface and not have it isolated to debug configuration.

```suggestion
        LOG(ERROR) << "Unable to initialize logging from handle.";
```